marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mastering/default/matching.sjs

```diff
@@ -20,28 +20,37 @@ const masteringStepLib = require("/data-hub/5/builtins/steps/mastering/default/l
 const requiredOptionProperties = ['matchOptions'];
 const emptySequence = Sequence.from([]);
 
-function filterContentAlreadyProcessed(content, summaryCollection) {
+function filterContentAlreadyProcessed(content, summaryCollection, collectionInfo) {
```

Comment:
I looked at checkOptions in lib.sjs, and I think it'd be worth documenting here that "collectionInfo" is expected to be produced by checkOptions. Otherwise it's difficult to know what this contains - gotta find the client who's calling this and where collectionInfo is coming from. 

In Intellij, when I asked it to auto-generate comments for the checkOptions function, it produced this, which may be useful to add as well so it's clear what's coming out of that function (as the name to me implies that it will either throw an error if there's a problem with the options or return nothing):

```
/**
 * 
 * @param content
 * @param options
 * @param filteredContent
 * @param reqOptProperties
 * @returns {{contentCollection: *, mergedCollection: *, notificationCollection: *, archivedCollection: *, auditingCollection: *}}
 */
```